crates/codegen/grammar/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+pub mod dsl;
```

Comment:
nit: looks like these are re-exported below. Should the `mod` declarations be private in this case?

Reply:
done

Reply:
I think one more remains: `dsl`.

Reply:
dsl is not exported below

Reply:
It is also not used anywhere outside this crate. Is this intended?